src/GitVersionCore/Logging/ConsoleAppender.cs

```diff
@@ -6,7 +6,7 @@ namespace GitVersion.Logging
     public class ConsoleAppender : ILogAppender
     {
         private readonly object _lock;
```

Comment:
```suggestion
        private readonly object lock;
```

Reply:
True. Another option would be to call it `locker`. :)

Reply:
yep, that's better, will change to locker. I have not yet pushed the changes